doc/pr2_tutorials/planning/src/doc/moveit_commander.rst

```diff
@@ -0,0 +1,47 @@
+MoveIt! Commander and Command Line Tool
+=======================================
+
+The `moveit_commander <http://wiki.ros.org/moveit_commander>`_ Python package offers wrappers for the functionality provided in MoveIt!. Simple interfaces are available for motion planning, computation of Cartesian paths, and pick and place.
+
+Command Line Tool
+-----------------
+
+The **moveit_commander** package also includes a command line tool for controlling the robot. 
+
+First, launch the MoveIt! demo interface: ::
+
+ roslaunch pr2_moveit_config demo.launch
+
+Now initiate the **moveit_commander** interface: ::
+
+ rosrun moveit_commander moveit_commander_cmdline.py
+  
+The command above will start a command line interface tool that allows you to connect to a running instance of the move_group node. The first command you should type is: ::
+
+ use <group name>
+
+This will connect to the move_group node for the group name you specified (in the PR2, for instance, you could connect to left_arm). You can now execute commands on that group. 
+This command, **current**, will show you the current state of your group: ::
+
+ current
+
+To record that state under a specific name you can simply type: ::
+
+ rec c
+ 
+This will remember the current joint values of the robot group under the name c. To get the robot to move, you could type, for example: :: 
```

Comment:
Could you move the description of the following code up here?
I know way too many people who would copy&paste the code before reading on, possibly with there own real robot connected... :/